### PR TITLE
Feature/dataeng 18205 investigate missing docs

### PR DESCRIPTION
**Pull request by Omhen**
In order to investigate why some documents are missing in postgress, we need to enable certain degree of logging we can easily query.
Logstash can now be configured as log appender, in parallel with stdout.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,3 +3,6 @@ gemspec
 gem 'simplecov', require: false, group: :test
 gem "codeclimate-test-reporter", "~> 1.0.0"
 gem 'rake'
+gem 'logging'
```
Comment (calonso) [inline Gemfile]: These two are already on the `gemspec`

File: lib/couch_tap/util.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module CouchTap
+  module Util
```
Comment (calonso) [inline lib/couch_tap/util.rb]: Why a module? Looks more like a class.

File: Gemfile (hunk 1 above)
Comment (Omhen) [reply Gemfile]: applied

File: lib/couch_tap/log_configurator.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                                 :ssl_enable => ssl_enable,
+                                 :type => "tcp",
+                                 :host => logstash_host,
+                                 :port => logstash_port)
+      Logging.logger.root.add_appenders('logstash')
+    end
+
+    def self.parse_level(level)
+      case level
+      when 'fatal' then :fatal
```
Comment (calonso) [inline lib/couch_tap/log_configurator.rb]: Nice one liners 👍

**Pull request merged**